packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -362,6 +367,17 @@ export const createRuntimeFactory = (): IRuntimeFactory => {
 					return aliasedDefault.get();
 				},
 			});
+			// id compressor isn't made available via the interface right now.
+			// We could revisit exposing the safe part of its API (IIdCompressor, not IIdCompressorCore) in a way
+			// that would avoid this instanceof check, but most customers shouldn't really have a need for it.
+			assert(runtime instanceof ContainerRuntime, "Expected to create a ContainerRuntime");
```

Comment:
i think we do have an internal set of container runtime options which could be used to pass that value to the runtime, rather than instance of check.

Reply:
yeah--that's another reasonable option, but would require some more id-compressor refactoring I don't really want to invest in since the ability to resize things isn't in the actual API surface, just a test export. I talked with Taylor and Noah about cluster size and it's pretty flexible, one could even use dynamic cluster sizing policies.